java/dagger/internal/codegen/compileroption/ProcessingEnvironmentCompilerOptions.java

```diff
@@ -404,7 +404,7 @@ public ImmutableList<String> aliases() {
      * How to report that an explicit binding in a subcomponent conflicts with an {@code @Inject}
      * constructor used in an ancestor component.
      */
-    EXPLICIT_BINDING_CONFLICTS_WITH_INJECT(WARNING, ERROR, NONE),
+    EXPLICIT_BINDING_CONFLICTS_WITH_INJECT(ERROR, WARNING, NONE),
```

Comment:
why would this matter?

Reply:
The order changes the default value -- the first value passed to the enum's constructor is the default value.